database/mssql/scripts/dbscripts/PSP_PIMS_S31_00/Alter Up/04_PSP_PIMS_PMBC_Tables_Build.sql

```diff
@@ -0,0 +1,111 @@
+-- Script generated by Aqua Data Studio Schema Synchronization for MS SQL Server 2016 on Tue Oct 05 16:47:03 PDT 2021
+-- Execute this script on:
+-- 		PIMS_S09_01/pmbc - This database/schema will be modified
+-- to synchronize it with MS SQL Server 2016:
+-- 		PIMS_S09_02/pmbc
+
+-- We recommend backing up the database prior to executing the script.
+
+SET XACT_ABORT ON
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+
+-- Create table pmbc.PMBC_PARCEL_POLYGON_FABRIC if not existing
+PRINT N'Create table pmbc.PMBC_PARCEL_POLYGON_FABRIC'
```

Comment:
@dfilteau this will wipe the PMBC table in all of our environments correct? can you let Peter spry know that whenever we release IS31 in a new environment he is going to need to re-execute the PMBC import?

Reply:
Correct. I asked Doug to make this change so PMBC table is created on local dev environments but not on live environments

Reply:
understood, my mistake.